frontend/client/views/pipeline/create.vue

```diff
@@ -32,6 +32,12 @@
                   </span>
                   <span>Add credentials</span>
                 </a>
+                <a class="button is-primary" v-on:click="showGitHubWebHookModal">
+                  <span class="icon">
+                    <i class="fa fa-certificate"></i>
```

Comment:
Would be awesome if you could change the icon for user experience 😃 see https://fontawesome.com/icons?d=gallery most of them are available

Reply:
Maybe https://fontawesome.com/icons/wrench?style=solid ?+

Reply:
Sure thing. :) Thanks! :)